jwst/stpipe/step.py

```diff
@@ -602,7 +602,7 @@ def get_reference_file_model(self, input_file, reference_file_type):
 
         Returns
         -------
-        reference_file_model : jwst_lib.models.ModelBase instance
+        reference_file_model : jwst.datamodels.ModelBase instance
             A model to access the contents of the reference file.
         """
         from jwst import datamodels
```

Comment:
You can change the import in this existing branch and push again to origin. It will show up as a separate commit. But yeah, we want to use relative imports.